test/unit/main_test.ts

```diff
@@ -40,3 +62,137 @@ describe('Recognizes invalid TypeScript inputs', () => {
     chai.expect(() => minifer.checkForErrors(program)).to.not.throw();
   })
 });
+
+describe('types', () => {
```

Comment:
You say you want to keep this code because it has already been written. But consider this: would you keep 200 lines of code in a method in main.ts, even though the method is never called, and it turns out you will never need it because you're following a different approach?

Less code is always better than more code, and these tests do no longer serve a purpose - they are not asserting that any code in this project is correct.